### ISSUE TITLE
Auto-suggest blocked by Header set X-Content-Type-Options "nosniff"

### ISSUE THREAD

**Issue by ajardin** [Flagbit/Magento-FACTFinder#219]
- FACT-Finder SDK version (see package.xml): 4.1.13
- Magento edition and version: EE 1.11.2
- PHP version: 5.6.35
- Expected behavior: The suggest feature working properly when the strict MIME type checking is enable.
- Actual behavior: `Refused to execute script from 'https://www.XXXXX.com/ff_suggest/proxy/suggest/?query=XXXXX' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.`
- Steps to reproduce: Add `Header set X-Content-Type-Options "nosniff"` in the server configuration.

I think it's related to another issue which is already closed (#86). The content-type sent by `FACTFinder_Suggest_ProxyController:: suggestAction()` is not appropriate and thus, blocked by the browser when the strict MIME type checking is enable. 

Based on the [RFC 4329](http://www.rfc-editor.org/rfc/rfc4329.txt), that controller should return **application/javascript** instead of **application/json**. There is already `format=JSONP` within the query string parameters so that seems consistent. And the suggestions are correctly displayed by the browser if that content-type is sent.

More details about that HTTP header:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Content-Type-Options

**Comment by mbraeuner** [comment]
fixed with #220 
available with Release 4.2.0
